src/operator/harbor.ts

```diff
@@ -366,7 +366,7 @@ async function processNamespace(namespace: string) {
     const projAdminMember: ProjectMember = {
       roleId: HarborRole.admin,
       memberGroup: {
-        groupName: 'team-admin',
+        groupName: 'all-teams-admin',
```

Comment:
what if I name my team `all-teams` ?

Reply:
hmm now I am thinking again about it and it may not be possible because all team groups start from team- prefix. If that is the case then let's resolve this conversation